homeronet/Plugin/Trek.cs

```diff
@@ -0,0 +1,77 @@
+﻿using homeronet.Client;
+using homeronet.Messages;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+using TrekQuotes = System.Collections.Generic.Dictionary<string, System.Collections.Generic.Dictionary<string, System.Collections.Generic.List<string>>>;
+
+namespace homeronet.Plugin {
+
+    public class Trek : IPlugin {
+        private List<string> _registeredCommands = new List<string>() { "trek" };
+        private TrekQuotes _quotes;
+        private Random _random = new Random();
+
+        public void Startup() {
+            using (StreamReader r = new StreamReader("Resources/trek.json")) {
+                // TODO: Strict contract
```

Comment:
For future reference, creating a contract is pretty easy with the jaysonz. you can copy-paste json in to VS and it'll auto generate the correct class for you.